tools/misc/split_coco.py

```diff
@@ -0,0 +1,107 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import argparse
+import os.path as osp
+
+import mmcv
+import numpy as np
+
+prog_description = '''K-Fold coco split.
```

Comment:
Is this necessary? seems that this variable is never used in this file

Reply:
It is referred to [mmcls](https://github.com/open-mmlab/mmclassification/blob/master/tools/kfold-cross-valid.py#L25) to add a description